src/kOS.Safe.Test/Serialization/TerminalFormatterTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Exceptions;
+using NUnit.Framework;
+using kOS.Safe.Serialization;
+
+namespace kOS.Safe.Test.Serialization
+{
+    [TestFixture]
+    public class TerminalFormatterTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+
+        }
+
+        [Test]
+        public void CanSerializeLexicons()
+        {
+            var lex = new Lexicon();
+            var nested = new Lexicon();
+
+            lex["key1"] = "value1";
+            lex["key2"] = 1;
+            lex["key3"] = nested;
+
+            nested["nested1"] = "nested1value";
+            nested["nested2"] = "nested2value";
+
+            Assert.AreEqual("LEXICON of 3 items:\n[\"key1\"]= value1\n[\"key2\"]= 1\n[\"key3\"]=" +
+                " LEXICON of 2 items:\n  [\"nested1\"]= nested1value\n  [\"nested2\"]= nested2value", Serialize(lex));
```

Comment:
I am sorry if i was unclear. The test fails on windows because of a newline character mismatch. I dont honestly care which we pick i would just like the test to not depend on OS in order to pass.